### PR TITLE
some design changes in board list

### PR DESCRIPTION
**Pull request by artemanufrij**
Signed-off-by: Artem Anufrij <artem.anufrij@live.de>

### CODE REVIEW

**Comment by artemanufrij** [comment]
* column title: color like `files`
* '...' - menu: opacity like `files`
* dot-icon: margin and size like `files`

**Comment by artemanufrij** [comment]
@juliushaertl done

File: css/style.css [hunk 1]
```diff
@@ -738,8 +741,20 @@ button.button-inline:hover {
 
 #boardlist td .app-popover-menu-utils {
 	float: right;
+	width: 30px;
+	display: inline;
+	position: relative;
+}
+
+#boardlist td .app-popover-menu-utils button {
+	opacity: 0.3;
```
Comment (pixelipo) [inline css/style.css]: I don't really see the point in using 4 different opacity levels. Difference between 0.3 and 0.5 or 0.5 and 0.7 is barely noticable and it doesn't give away any additional message to the user.

I'm working on cleaning up opacity definitions in nextcloud/server and the idea is to use 0.5 and 1 for most of the situations, and I think this is a perfect example. Triple-dot button should  have 50% opacity normally and 100% when hovered/focused/selected wherever it is used.

File: css/style.css [hunk 2]
```diff
@@ -738,8 +741,20 @@ button.button-inline:hover {
...
 }
 
+#boardlist td .app-popover-menu-utils:hover button {
+	opacity: 0.7;
```
Comment (pixelipo) [inline css/style.css]: Part of the above comment - `opacity:1`

In the long term, this should all be moved into `.button-inline` and `.button-inline:hover, .button-inline: focus`.

File: templates/part.boardlist.php [hunk 3]
```diff
@@ -24,7 +24,7 @@
 			<td>
 				<div class="hint"></div>
 				<div class="app-popover-menu-utils" ng-if="b.deletedAt == 0">
-					<button class="icon icon-more button-inline" title="More actions"></button>
+					<button class="icon icon-more button-inline" title="<?php p($l->t('More actions')); ?>"></button>
```
Comment (juliushaertl) [inline templates/part.boardlist.php]: :+1:

**Comment by juliushaertl** [comment]
Great @artemanufrij. Thank you :)

File: templates/part.boardlist.php (hunk 3 above)
Comment (pixelipo) [inline templates/part.boardlist.php]: Great catch!

File: css/style.css (hunk 2 above)
Comment (artemanufrij) [inline css/style.css]: @pixelipo currently it has same behavior like files

Comment (pixelipo) [inline css/style.css]: That doesn't make it correct, @artemanufrij - plus it gives us a third opacity behaviour for the same button:
- triple-dot in navigation bar: 0.5->1
- triple-dot on card: 0.25->1
- triple-dot on board list: 0.3->0.7

I will make a PR to Files to fix it there as well.

Comment (artemanufrij) [inline css/style.css]: @pixelipo you are right... i focused only board list

**Pull request merged**